packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -64,7 +64,7 @@
 
 #### docs
 
-- Update whitelist to allowlist ([PR #2323](https://github.com/forcedotcom/salesforcedx-vscode/pull/2323))
+- Update allowlist to allowlist ([PR #2323](https://github.com/forcedotcom/salesforcedx-vscode/pull/2323))
```

Comment:
This is a bit confusing

Reply:
@xyc Would it be better to rephrase as "replaced non-inclusive content"?

Reply:
Yes that rephrasing would be good

Reply:
Done!